### PR TITLE
Fix naming and structure, follow conventions

### PR DESCRIPTION
**Pull request by karlhorky**
Fix some naming, structure and conventions things

@Josehower @ProchaLu @Eprince-hub you should probably all look at all changes in the Files Changed tab and then approve + merge

Some of the issues highlighted:

Screenshot 2023-01-26 at 19 03 52

---

Screenshot 2023-01-26 at 19 05 14

---

Screenshot 2023-01-26 at 19 05 24

---

Screenshot 2023-01-26 at 19 05 36

---

Screenshot 2023-01-26 at 19 05 53

---

Screenshot 2023-01-26 at 19 07 04

### CODE REVIEW

**Review by ProchaLu — APPROVED** [review]
lgtm 👍

**Review by Eprince-hub — APPROVED** [review]
Looks good ✅

File: app/(auth)/register/page.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { redirect } from 'next/navigation';
 import { getValidSessionByToken } from '../../../database/sessions';
 import RegisterFrom from './RegisterForm';
 
-export default async function Login() {
+export default async function LoginPage() {
```
Comment (Josehower) [inline app/(auth)/register/page.tsx:6]: RegisterPage here

File: app/animal-management-naive-dont-copy/delete/[animalId]/head.jsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import { getAnimalById } from '../../../../database/animals';
 
-export default async function Head(props) {
+export default async function AnimalManagementNaiveDontCopyAnimalDeleteHead(
+  props,
+) {
```
Comment (Josehower) [inline app/animal-management-naive-dont-copy/delete/[animalId]/head.jsx:5]: hmm maybe

AnimalManagementNaiveDontCopyAnimalDeleteHeadById ??

this name is not taking into account the [animalId]

File: app/animal-management-naive-dont-copy/delete/[animalId]/page.jsx [hunk 3]
```diff
@@ -2,7 +2,9 @@ import Image from 'next/image';
 import { notFound } from 'next/navigation';
 import { deleteAnimalById } from '../../../../database/animals';
 
-export default async function SingleAnimal(props) {
+export default async function AnimalManagementNaiveDontCopyAnimalDeletePage(
```
Comment (Josehower) [inline app/animal-management-naive-dont-copy/delete/[animalId]/page.jsx:5]: similar change as here https://github.com/upleveled/next-13-app-dir-demo-project/pull/5/files#r1090370825

File: app/animal-management-naive-dont-copy/read/[animalId]/head.jsx [hunk 4]
```diff
@@ -1,6 +1,8 @@
 import { getAnimalById } from '../../../../database/animals';
 
-export default async function Head(props) {
+export default async function AnimalManagementNaiveDontCopyAnimalReadHead(
```
Comment (Josehower) [inline app/animal-management-naive-dont-copy/read/[animalId]/head.jsx:3]: this name is not taking into account [animalId]

https://github.com/upleveled/next-13-app-dir-demo-project/pull/5/files#r1090370825

File: app/animal-management-naive-dont-copy/read/[animalId]/page.jsx [hunk 5]
```diff
@@ -2,7 +2,9 @@ import Image from 'next/image';
 import { notFound } from 'next/navigation';
 import { getAnimalById } from '../../../../database/animals';
 
-export default async function SingleAnimal(props) {
+export default async function AnimalManagementNaiveDontCopyAnimalReadPage(
```
Comment (Josehower) [inline app/animal-management-naive-dont-copy/read/[animalId]/page.jsx:5]: this name is not taking into account [animalId]

[#5 (files)](https://github.com/upleveled/next-13-app-dir-demo-project/pull/5/files#r1090370825)

File: app/animal-management-naive-dont-copy/update/[animalId]/head.jsx [hunk 6]
```diff
@@ -1,6 +1,8 @@
 import { getAnimalById } from '../../../../database/animals';
 
-export default async function Head(props) {
+export default async function AnimalManagementNaiveDontCopyUpdateAnimalHead(
```
Comment (Josehower) [inline app/animal-management-naive-dont-copy/update/[animalId]/head.jsx:3]: this name is not taking into account [animalId]

[#5 (files)](https://github.com/upleveled/next-13-app-dir-demo-project/pull/5/files#r1090370825)

File: app/animal-management-naive-dont-copy/update/[animalId]/page.jsx [hunk 7]
```diff
@@ -2,7 +2,9 @@ import Image from 'next/image';
 import { notFound } from 'next/navigation';
 import { updateAnimalById } from '../../../../database/animals';
 
-export default async function Animal(props) {
+export default async function AnimalManagementNaiveDontCopyUpdateAnimalPage(
```
Comment (Josehower) [inline app/animal-management-naive-dont-copy/update/[animalId]/page.jsx:5]: this name is not taking into account [animalId]

[#5 (files)](https://github.com/upleveled/next-13-app-dir-demo-project/pull/5/files#r1090370825)

File: app/(auth)/register/page.tsx (hunk 1 above)
Comment (karlhorky) [reply app/(auth)/register/page.tsx]: [`5500985` (#5)](https://github.com/upleveled/next-13-app-dir-demo-project/pull/5/commits/5500985117b07d77bd63003bf6346f0ea54885a9)

File: app/animals/page.tsx [hunk 8]
```diff
@@ -1,9 +1,9 @@
 import Image from 'next/image';
 import Link from 'next/link';
 import { getAnimals } from '../../database/animals';
-import styles from './animals.module.scss';
+import styles from './page.module.scss';
```
Comment (Josehower) [inline app/animals/page.tsx:4]: maybe Page.module.scss to follow the capital pattern in other files

File: app/animal-management-naive-dont-copy/delete/[animalId]/head.jsx (hunk 2 above)
Comment (karlhorky) [reply app/animal-management-naive-dont-copy/delete/[animalId]/head.jsx:5]: there's no other delete page yet, so `AnimalManagementNaiveDontCopyAnimalDeleteHead` is unique

File: app/animal-management-naive-dont-copy/delete/[animalId]/page.jsx (hunk 3 above)
Comment (karlhorky) [reply app/animal-management-naive-dont-copy/delete/[animalId]/page.jsx:5]: there's no other delete page yet, so `AnimalManagementNaiveDontCopyAnimalDeleteHead` is unique

File: app/head.tsx [hunk 9]
```diff
@@ -1,4 +1,4 @@
-export default function Head() {
+export default function HomeHead() {
```
Comment (Josehower) [inline app/head.tsx:1]: what about RootHead this is going to apply to any nested page without head

File: app/loading.tsx [hunk 10]
```diff
@@ -1,3 +1,3 @@
-export default function Loading() {
+export default function HomeLoading() {
```
Comment (Josehower) [inline app/loading.tsx:1]: what about RootLoading this is going to apply to any nested page without Loading

File: app/page.tsx [hunk 11]
```diff
@@ -1,8 +1,8 @@
 import Image from 'next/image';
 import zebra from '../public/zebra.jpeg';
-import styles from './home.module.scss';
+import styles from './page.module.scss';
```
Comment (Josehower) [inline app/page.tsx:3]: capitalization

File: app/animal-management-naive-dont-copy/read/[animalId]/page.jsx (hunk 5 above)
Comment (karlhorky) [reply app/animal-management-naive-dont-copy/read/[animalId]/page.jsx:5]: - it's `Animal` vs `Animals`, so the id is implied from the name

File: app/animal-management-naive-dont-copy/update/[animalId]/head.jsx (hunk 6 above)
Comment (karlhorky) [reply app/animal-management-naive-dont-copy/update/[animalId]/head.jsx:3]: - it's `Animal` vs `Animals`, so the id is implied from the name

**Review by Josehower — CHANGES_REQUESTED** [review]
Is almost there i think is a pretty good PR. Just we need to polish some naming

File: app/page.tsx (hunk 11 above)
Comment (karlhorky) [reply app/page.tsx:3]: capitalization already matches the page filename - that's how CSS modules are done most commonly

File: app/animal-management-naive-dont-copy/update/[animalId]/head.jsx (hunk 6 above)
Comment (Josehower) [reply app/animal-management-naive-dont-copy/update/[animalId]/head.jsx:3]: kk

File: app/animals/page.tsx (hunk 8 above)
Comment (karlhorky) [reply app/animals/page.tsx:4]: capitalization already matches the page filename - that's how CSS modules are done most commonly

File: app/animal-management-naive-dont-copy/update/[animalId]/page.jsx (hunk 7 above)
Comment (karlhorky) [reply app/animal-management-naive-dont-copy/update/[animalId]/page.jsx:5]: - it's `Animal` vs `Animals`, so the id is implied from the name

File: app/loading.tsx (hunk 10 above)
Comment (karlhorky) [reply app/loading.tsx]: 6c3f0c4379850d9fb3dd3db0f3743d88ee0b9070

File: app/head.tsx (hunk 9 above)
Comment (karlhorky) [reply app/head.tsx]: 4e5f7c05b8b4e698c9c36a52e64b821436501435

**Pull request merged**